sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -151,6 +151,14 @@ ExecutionContext GetContext (SessionId sessionId)
 			throw new ArgumentException ($"Invalid Session: \"{id}\"", nameof (sessionId));
 		}
 
+		bool SetContext (SessionId sessionId, ExecutionContext executionContext, out ExecutionContext previousExecutionContext)
```

Comment:
Should this be called `ChangeContext` or `SwitchContext`?

Reply:
changed to UpdateContext